updater/metadata_fetchers/nvd/nvd.go

```diff
@@ -224,3 +230,22 @@ func getHashFromMetaURL(metaURL string) (string, error) {
 
 	return "", errors.New("invalid .meta file format")
 }
+
+// scoreToPriority converts the CVSS Score (0.0 - 10.0) into the user-friendy
+// types.Priority using the usually reference correlation table, extended
+// for the Negligible level.
+func scoreToPriority(score float64) types.Priority {
```

Comment:
Yeah, thanks. That's what I used indirectly but didn't specify any link. Let me edit the comment. I still believe that `< 1.0` is better than `= 0.0` for the _Negligible_ Severity as 0.0 is virtually non-existent. If you prefer keeping `= 0.0`, I'm open to that.